backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/SimpleConnectionPool.java

```diff
@@ -57,6 +57,12 @@ public static RequiresRetries builder() {
             return retries -> initialDelay -> new Configuration(retries, initialDelay);
         }
 
+        public static Configuration from(org.apache.commons.configuration2.Configuration configuration) {
+            return builder()
+                .retries(configuration.getInt("connection.pool.retries", 10))
+                .initialDelay(Duration.ofMillis(configuration.getLong("connection.pool.min.delay.ms", 100)));
```

Comment:
no named variable?

Reply:
Why?

The builder method dis-ambiguate the context...